conda/meta.yaml

```diff
@@ -19,14 +19,10 @@ build:
 
 test:
   requires:
-    - pytest
```

Comment:
Just curious as to why all of this is disappearing.

Reply:
So Tiago made a conda metapackage ts-conda-build which installs a number of packages in our stack. This allows us to just list that package as a dependency.